xtext-website/documentation/310_eclipse_support.md

```diff
@@ -236,6 +236,13 @@ public void createReferenceType(Issue issue, IssueResolutionAcceptor acceptor) {
 
 Hence, there is the [ISyntaxErrorMessageProvider]({{site.src.xtext_core}}/org.eclipse.xtext/src/org/eclipse/xtext/parser/antlr/ISyntaxErrorMessageProvider.java) to assign issue codes to syntactical errors.
 
+## Auto Editing {#autoediting}
+Xtext-based editors automatically assist the user by inserting/deleting certain elements on typing. E.g. inserting/removing closing single quotes, double quotes, parenthesis, square brackets or curly braces when the user inserts/removes the opening ones. Moreover, the auto-indentation functionality ensures the indentation-awareness of new lines: after hitting the `ENTER` key e.g. in a block enclosed by curly braces the cursor is automatically placed on the indented position in the subsequent new line.
```

Comment:
is "elements" effectively "text"?

Reply:
`text` is really a better choice here. I updated the PR.